### DISCUSSION TITLE
20240229 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by josecelano** [nautilus-cyberneering/.github#84] [Meetings]
- Time:  9:30 in Canary Islands (WET (UTC+0)).
- Time: 10:30 in Vaduz (CET (UTC+1)).

Location:
<https://meeting.blockfinance-eco.li/NautilusCyberneeringWeeklyDevMeeting>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- CGarnham(@da2ce7)  - 
- JCelano(@josecelano) - x
- CBosse(@cgbosse) - x
- MMelchor(@mario-nt) - x
- GByrne(@grmbyrn) - x
- MVanDijke(@WarmBeer) - 

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by josecelano** [comment]
### Torrust Tracker

- https://github.com/torrust/torrust-tracker/pull/716
- https://github.com/torrust/torrust-tracker/issues/714
- https://github.com/torrust/torrust-tracker/issues/704
- https://github.com/torrust/torrust-tracker/issues/528
- https://github.com/torrust/torrust-tracker/issues/155

Discussion:

- https://github.com/torrust/torrust-tracker/issues/708

### Torrust Index

- https://github.com/torrust/torrust-index/pull/500
- https://github.com/torrust/torrust-index/issues/455
- https://github.com/torrust/torrust-index/issues/503
- https://github.com/torrust/torrust-index/issues/488
- https://github.com/torrust/torrust-index/pull/491

Hey, @da2ce7 @WarmBeer I think I'm going to implement this solution:

https://github.com/torrust/torrust-index/issues/469#issuecomment-1935695015

to import statistics from the Tracker unless you think it's not a good idea.

Discussion:

- https://github.com/torrust/torrust-index/issues/498
- https://github.com/torrust/torrust-index/issues/499

### Torrust Index GUI

- https://github.com/torrust/torrust-index-gui/pull/488
- https://github.com/torrust/torrust-index-gui/issues/486
- https://github.com/torrust/torrust-index-gui/issues/483

### Torrust Website

PR review:

- https://github.com/torrust/torrust-website/pull/141
- https://github.com/torrust/torrust-website/pull/142

### Operative 

Meetings with @cgbosse and other stuff:

- Preparing one-on-ones
- Internships for students

**Comment by mario-nt** [comment]
### GUI

- https://github.com/torrust/torrust-index-gui/issues/229
- https://github.com/torrust/torrust-index-gui/issues/457

### Index
	
- https://github.com/torrust/torrust-index/issues/448
	

### Discussions
Read discussions:

> - https://github.com/torrust/torrust-index/issues/469#issuecomment-1952237221
> - https://github.com/torrust/torrust-tracker/discussions/708
> - https://github.com/torrust/torrust-index/discussions/498
> - https://github.com/torrust/torrust-index-gui/issues/417#issuecomment-1966440247

### Training/Research

- Cypress best practices, aliases, tasks vs commands...

**Comment by grmbyrn** [comment]
## Torrust Website
* [https://github.com/torrust/torrust-website/pull/141](https://github.com/torrust/torrust-website/pull/141)
* [https://github.com/torrust/torrust-website/pull/142](https://github.com/torrust/torrust-website/pull/142)
* [skeleton.dev](https://www.skeleton.dev/)
* [Intersection Observer](https://www.youtube.com/watch?v=2IbRtjez6ag&ab_channel=WebDevSimplified)
* [https://github.com/torrust/torrust-website/issues/139](https://github.com/torrust/torrust-website/issues/139)
* [https://github.com/torrust/torrust-website/issues/143](https://github.com/torrust/torrust-website/issues/143)

## Practice
* React through [projects](https://github.com/grmbyrn/github-api-finder)
* Practice SvelteKit through Frontend Mentor [projects](https://github.com/grmbyrn/faq-accordion-main)
